backend/ohq/models.py

```diff
@@ -268,6 +269,89 @@ class Question(models.Model):
     tags = models.ManyToManyField(Tag, blank=True)
 
 
+class QueueStatistic(models.Model):
+    """
+    Statistics related to a queue
+    """
+
+    # add new metrics/statistics
+    METRIC_HEATMAP_WAIT = "HEATMAP_AVG_WAIT"
+    METRIC_HEATMAP_QUESTIONS_PER_TA = "HEATMAP_QUESTIONS_PER_TA"
+    METRIC_AVG_WAIT = "AVG_WAIT"
+    METRIC_NUM_ANSWERED = "NUM_ANSWERED"
+    METRIC_STUDENTS_HELPED = "STUDENTS_HELPED"
+    METRIC_AVG_TIME_HELPING = "AVG_TIME_HELPING"
+    METRIC_LIST_WAIT_TIME_DAYS = "LIST_WAIT_TIME_DAYS"
+    METRIC_CHOICES = [
+        (METRIC_HEATMAP_WAIT, "Average wait-time heatmap"),
+        (METRIC_HEATMAP_QUESTIONS_PER_TA, "Questions per TA heatmap"),
+        (METRIC_AVG_WAIT, "Average wait-time"),
+        (METRIC_NUM_ANSWERED, "Number of questions answered per week"),
+        (METRIC_STUDENTS_HELPED, "Students helped per week"),
+        (METRIC_AVG_TIME_HELPING, "Average time helping students"),
+        (METRIC_LIST_WAIT_TIME_DAYS, "List of wait times per day"),
+    ]
+
+    # for specific days during the week - used for heatmap and graphs where day is x-axis
+    DAY_MONDAY = 0
```

Comment:
Maybe we should change Sunday to the first day of the week? I think having Monday first could cause some confusion

Reply:
Should we also just follow django ORM conventions by using 1-7?

Reply:
Yep will reorder the days to be Sunday = 1, Monday = 2, etc.